.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
It's a specific configuration linked to your own environment. It should be done inside your global gitignore configuration, not in the configuration of this project ;) https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore